processor/k8sattributesprocessor/metadata.yaml

```diff
@@ -196,6 +196,27 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    otelsvc_k8s_deployment_updated:
+      enabled: true
+      description: Number of deployment update events received
+      unit: "1"
+      sum:
+        value_type: int
+        monotonic: true
+    otelsvc_k8s_deployment_added:
+      enabled: true
```

Comment:
Please turn those off by default for now, and make a separate PR to enable those by default as a breaking change.